.eslintrc

```diff
@@ -2,7 +2,10 @@
   "env": {
     "jest": true
   },
-  "extends": "airbnb",
+  "extends": [
+    "airbnb",
```

Comment:
should we be also getting rid of `airbnb`?

Reply:
I don't think so, it has some useful rules.